python/tvm/micro/testing/evaluation.py

```diff
@@ -83,6 +83,7 @@ def create_aot_session(
     params,
     build_dir=Path(tempfile.mkdtemp()),
     tune_logs=None,
+    timeout_override=None,
```

Comment:
nit: units e.g. timeout_sec_override

Reply:
This isn't a float - this should be a `tvm.micro.project_api.server.TransportTimeouts` object (`timeout_override` is always this type in our codebase). We could add typing hints to this file, but I think that's OOS for this PR.